src/schema/v2/homeView/getSectionsForUser.ts

```diff
@@ -23,10 +24,14 @@ export async function getSectionsForUser(
 
   const me = await meLoader()
 
-  let sections: HomeViewSection[] = []
+  let sections: HomeViewSectionOrResolver[] = []
 
   if (me.type === "Admin") {
     sections = [
+      featuredCollection(
+        "curators-picks-emerging-app",
```

Comment:
Here I pass heroUnit ID and marketingCollection ID. Eigen hardcodes marketingColection ID, then fetches all heroUnits and picks only the one with corresponding marketing collection ID. I can imaging that in future we can have another heroUnit corresponding to another marketing collection. It will be easy to support by adding another `featuredCollection(heroUnitID, marketingCollectionID)`.